pkg/webui/console/store/reducers/events.js

```diff
@@ -76,6 +76,12 @@ const addEvent = (state, event) => {
     return {}
   }
 
+  // When requesting historical events (with tail / after) it's possible that we
+  // still have some of those in the store, so we don't add them again.
+  if (event.time < streamStartTime && events.find(e => e.unique_id === event.unique_id)) {
+    return {}
+  }
+
```

Comment:
It is currently necessary to do this since the `after` prop in the event stream request is not 100% accurate likely due to minimal differences in the clocks. This leads to the very last historical event still coming through.

Once this is resolved we can remove this check. I've already limited the costly `events.find()` to historical events only.

cc @htdvisser